pom.xml

```diff
@@ -177,6 +177,23 @@
                     </dependency>
                 </dependencies>
             </plugin>
+            <plugin>
+                <artifactId>maven-enforcer-plugin</artifactId>
+                <executions>
+                    <execution>
+                        <id>display-info</id>
+                        <configuration>
+                            <rules>
+                                <requireUpperBoundDeps>
+                                    <excludes combine.children="append">
+                                        <exclude>org.apache.commons:commons-lang3</exclude>
```

Comment:
Exclude commons-lang3 as it breaks requireUpperBoundDeps enforcer rule.  jenkins-test-harness is test scoped thus this conflict can be ignored.

Error with exclude:
```
Require upper bound dependencies error for org.apache.commons:commons-lang3:3.3.2 paths to dependency are:
+-org.jenkins-ci.plugins:crowd2:2.0.0-SNAPSHOT
  +-com.atlassian.crowd:crowd-integration-client-rest:2.8.8
    +-com.atlassian.crowd:crowd-integration-api:2.8.8
      +-org.apache.commons:commons-lang3:3.3.2
and
+-org.jenkins-ci.plugins:crowd2:2.0.0-SNAPSHOT
  +-org.jenkins-ci.main:jenkins-test-harness:2.38
    +-org.jenkins-ci.main:jenkins-test-harness-htmlunit:2.18-1
      +-org.apache.commons:commons-lang3:3.4
and
+-org.jenkins-ci.plugins:crowd2:2.0.0-SNAPSHOT
  +-com.atlassian.crowd:crowd-integration-client-rest:2.8.8
    +-com.atlassian.crowd:crowd-integration-api:2.8.8
      +-com.atlassian.crowd:embedded-crowd-api:2.8.8
        +-org.apache.commons:commons-lang3:3.3.2
```